### DISCUSSION TITLE
INQUIRY

### DISCUSSION THREAD

**Discussion by odetteg** [sandbox-bio/sandbox.bio#9] [Q&A] [answered]
In the sars-cov amplicon sequence analysis, running the command "cat pileup.txt | ivar variants -r $REF_FASTA -g $REF_GFF -p variants.tsv -m 10" returns GFF file is not in GFF3 file format!

**Comment by niemasd** [comment, marked as answer]
Great question! This is a warning that iVar Variants prints because of the format of the primer BED file we have provided (there are different versions of the GFF file format), but this warning can be safely ignored: it doesn't impact the results

**Reply by robertaboukhalil** [reply]
Thanks @niemasd!
